components/server/src/workspace/workspace-starter.ts

```diff
@@ -867,6 +867,19 @@ export class WorkspaceStarter {
                 featureFlags = featureFlags.concat(["protected_secrets"]);
             }
 
+            if (
+                // ConfigCat already has feature flag named `persistent_volume_claim` that controls if we show
+                // the option to enable PVC for project prebuilds. So have to use a different name here.
+                await getExperimentsClientForBackend().getValueAsync("user_pvc", false, {
```

Comment:
webapp team, when reviewing, can you double check that this is indeed the only place where I need to add this code?

Reply:
This change can actually go away, the other two are fine! :+1:

Reply:
Hm. Are you sure? Since I originally tested with this change only and it worked as expected. So want to double check before I remove this code. :thinking:

Reply:
> Are you sure?

Yes. That's the core of the [PR from last week](https://github.com/gitpod-io/gitpod/pull/13117): `persistent_volume_claim` is a "workspace persistent" feature, so it should never be required to modify on start, only on creation.

This PR just changes the source from where we get the info from, so we should be able to simply follow all occurrences of `user.featureFlags?.permanentWSFeatureFlags?.includes("persistent_volume_claim")` and that check there (as you already did).

Reply:
@geropl ah, good to know. thank you! code removed. PTAL